packages/cdk8s-cli/.projenrc.js

```diff
@@ -27,12 +30,6 @@ const project = new TypeScriptLibraryProject({
   ...common.options,
 });
 
-// add @types/node as a regular dependency since it's needed to during "import"
-// to compile the generated jsii code.
-project.addDependencies({
-  '@types/node': Semver.caret(project.devDependencies['@types/node'].replace(/^\^/, ''))
```

Comment:
@eladb I just moved this to be a regular dependency. The dev dependency version should align just like any other - or am i missing something?